python/src/iceberg/schema.py

```diff
@@ -203,32 +210,32 @@ def after_map_value(self, value: NestedField) -> None:
     @abstractmethod
     def schema(self, schema: Schema, struct_result: T) -> T:
         """Visit a Schema"""
-        ...
+        ...  # pragma: no cover
 
     @abstractmethod
     def struct(self, struct: StructType, field_results: List[T]) -> T:
         """Visit a StructType"""
-        ...
+        ...  # pragma: no cover
 
     @abstractmethod
     def field(self, field: NestedField, field_result: T) -> T:
         """Visit a NestedField"""
-        ...
+        ...  # pragma: no cover
 
     @abstractmethod
     def list(self, list_type: ListType, element_result: T) -> T:
         """Visit a ListType"""
-        ...
+        ...  # pragma: no cover
 
     @abstractmethod
     def map(self, map_type: MapType, key_result: T, value_result: T) -> T:
         """Visit a MapType"""
-        ...
+        ...  # pragma: no cover
 
     @abstractmethod
     def primitive(self, primitive: PrimitiveType) -> T:
         """Visit a PrimitiveType"""
-        ...
+        ...  # pragma: no cover
```

Comment:
What does this do?

Reply:
This tells coverage to not include this line in the coverage % calculations.